packages/react/src/components/List/List.tsx

```diff
@@ -83,6 +85,7 @@ class List extends AutoControlledComponent<WithAsProp<ListProps>, ListState> {
     debug: PropTypes.bool,
     items: customPropTypes.collectionShorthand,
     selectable: PropTypes.bool,
+    navigable: PropTypes.bool,
```

Comment:
I don't think it is valid to create `<List selectable navigable ... />`
I would go with someting like this:
```jsx
selectable: customPropTypes.every([customPropTypes.disallow(['navigable']), PropTypes.bool]),
navigable: customPropTypes.every([customPropTypes.disallow(['selectable']), PropTypes.bool]),
```